hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -110,18 +104,18 @@ public void injectProperties() {
     
 	/**
      * This method initializes the Validators. 
-     * If there was an error in intializing jwt processor, HNSecure server will not start
+     * If there was an error in loading validators, HNSecure server will not start
+     * Currently only TokenValidator is throwing error. 
      */
     public void loadValidator() {
-    	v2PayloadValidator = new V2PayloadValidator();
     	try {
-    		validateAccessToken = new ValidateAccessToken();
-    	} catch(MalformedURLException t) {
-    		logger.error("Error in starting server: ", t);
-    		System.exit(0);
-    	}
-
-    			
+    		// This syntax will make sure validation is happening in the order of First Token validation then Payload validation then  ValidatorImpl
+			validator = new TokenValidator(new PayLoadValidator(new ValidatorImpl()));
+		} catch (MalformedURLException e) {
+			logger.error("Error in server startup: ", e);
+			logger.error("Stopping HNSecure Server. ");
```

Comment:
I don't see anything that would actually cause the server startup to fail.

Reply:
My thoughts are.. if the values in Application.properties are incorrect and initialization fails, then we should not start the server.

Reply:
Sorry I wasn't clear...in this branch there's actually no System.exit statement that will shut down the server on a MalformedURLException.

Reply:
ahh ... understand now :). will add it